tools/bridge/bridge/confirmation_sender.py

```diff
@@ -80,30 +89,65 @@ def __init__(
             )
         ]
 
-    @sender_retry
+        self.is_parity = self.w3.clientVersion.startswith("Parity")
```

Comment:
should we just raise a `SetupException` if it's not Parity? It's the only client that works for the trustlines chain.

Reply:
This would break at least the tests. Maybe others want to use the bridge in a different setup. I wouldn't want to prevent them doing that.